+++
bibtex_type = "article"
author="Wenson L, Heldin J, Martin M, Erbilgin Y, Salman B, Sundqvist A, Schaal W, Sandbaumhüter FA, Jansson ET, Chen X, Davidsson A, Stenerlöw B, Espinoza JA, Lindström M, Lennartsson J, Spjuth O, and Söderberg O."
title="Precise mapping of single-stranded DNA breaks by sequence-templated erroneous DNA polymerase end-labelling"
journal="Nature Communications"
year="2025"
date="2025-08-04T00:00:00+02:00"
volume="16"
number="7130"
preprint = false
pages=""
abstract="The ability to analyze whether DNA contains lesions is essential in identifying mutagenic substances. Currently, the detection of single-stranded DNA breaks (SSBs) lacks precision. To address this limitation, we develop a method for sequence-templated erroneous end-labelling sequencing (STEEL-seq), which enables the mapping of SSBs. The method requires a highly error-prone DNA polymerase, so we engineer a chimeric DNA polymerase, Sloppymerase, capable of replicating DNA in the absence of one nucleotide. Following the omission of a specific nucleotide (e.g., dATP) from the reaction mixture, Sloppymerase introduces mismatches directly downstream of SSBs at positions where deoxyadenosine should occur. This mismatch pattern, coupled with the retention of sequence information flanking these sites, ensures that the identified hits are bona fide SSBs. STEEL-seq is compatible with a variety of sequencing technologies, as demonstrated using Sanger, Illumina, PacBio, and Nanopore systems. Using STEEL-seq, we determine the SSB/base pair frequency in the human genome to range between 0.7 and 3.8 × 10−6 with an enrichment in active promoter regions."
doi="10.1038/s41467-025-62512-4"
url_html=""
+++


