+++
title = "SciPipe workflow library reviewed in Nature Methods perspective"
description = "TBC"
date = "2022-06-14T17:14:00+01:00"
taxonomies = "blogging"
teaser_image = "/img/natmet-workflows.png"
+++

We are a bit late on this, but we are happy to inform that the workflow
tool/library we have developed and used in the group, [SciPipe](https://pharmb.io/tool/scipipe/),
was recently reviewed in a workflow review in the journal Nature Methods:

- Wratten, L., Wilm, A. & Göke, J. *Reproducible, scalable, and shareable
  analysis pipelines with bioinformatics workflow managers*. _Nat Methods_ 18,
  1161–1168 (2021). https://doi.org/10.1038/s41592-021-01254-9

We additionally think this is probably the one workflow review that so far does
the best job in describing and characterizing the most popular tools in the
field, their strong and weak points etc, and this includes SciPipe. Probably
our favourite workflow review so far. Warmly recommended reading!

In fact, SciPipe was also briefly mentioned in another recent article, in the
Nature journal Scientific Reports. This article provides some useful further
technical insights into the task of writing workflows in a few different
systems:

- Ahmed, A.E., Allen, J.M., Bhat, T. et al. Design considerations for workflow
  management systems use in production genomics research and the clinic. Sci
  Rep 11, 21680 (2021). https://doi.org/10.1038/s41598-021-99288-8

See also our [last blog post about SciPipe](/blog/scipipe-published-in-gigascience/), that gives some overview
comments about the project.

## Our favorite workflow papers

While at the topic of workflow reviews, we might as well include a selection of
our earlier favourite workflow papers, to make up a whole little reading list
for anyone interested.

Apart from the articles above, the following papers have been recommended in
some of our courses, as further reading material on workflows:

- Spjuth, O., Capuccini, M., Carone, M., Larsson, A., Schaal, W., Novella, J.
  A., ... & Lampa, S. (2021).
  [Approaches for containerized scientific workflows in cloud environments with applications in life science. F1000Research](https://doi.org/10.12688/f1000research.53698.1),
  10(513), 513.
- Grüning, B. A., Lampa, S., Vaudel, M., & Blankenberg, D. (2019).
- [Software engineering for scientific big data analysis](https://doi.org/10.1093/gigascience/giz054).
  GigaScience, 8(5), giz054.
- Lampa, S., Dahlö, M., Alvarsson, J., & Spjuth, O. (2019).
  [SciPipe: A workflow library for agile development of complex and dynamic bioinformatics pipelines. GigaScience](https://doi.org/10.1093/gigascience/giz044),
  8(5), giz044.
- Leipzig, J. (2017).
  [A review of bioinformatic pipeline frameworks](https://doi.org/10.1093/bib/bbw020).
  Briefings in bioinformatics, 18(3), 530-536.
- Cohen-Boulakia, S., Belhame, K., Collin, O., Chopard, J., Froidevaux, C.,
  Gaignard, A., ... & Blanchet, C. (2017).
  [Scientific workflows for computational reproducibility in the life sciences: Status, challenges and opportunities](https://doi.org/10.1016/j.future.2017.01.012).
  Future Generation Computer Systems, 75, 284-298.
- Spjuth, O., Bongcam-Rudloff, E., Hernández, G. C., Forer, L., Giovacchini,
  M., Guimera, R. V., ... & Vassilev, D. (2015).
  [Experiences with workflows for automating data-intensive bioinformatics](https://doi.org/10.1186/s13062-015-0071-8).
  Biology direct, 10(1), 1-12.


## SciPipe Links

- [Main website and documentation](https://scipipe.org)
- [GitHub repository](https://github.com/scipipe/scipipe)
- [Publication in GigaScience](https://pharmb.io/publication/2019-scipipe/)
- [Publication in IEEE / CiSE](https://doi.org/10.1109/MCSE.2019.2907814)
